river/tree/__init__.py

```diff
@@ -72,4 +73,5 @@
     "LabelCombinationHoeffdingTreeClassifier",
     "SGTClassifier",
     "SGTRegressor",
+    "MondrianTreeClassifier",
```

Comment:
Let's keep the imports and the placement of class names inside `__all__` in alphabetical order.